auth.go

```diff
@@ -38,3 +38,38 @@ func (api *Client) SendAuthRevokeContext(ctx context.Context, token string) (*Au
 
 	return api.authRequest(ctx, "auth.revoke", values)
 }
+
+type listTeamsResponse struct {
+	Teams []Team `json:"teams"`
+	SlackResponse
+}
+
+type ListTeamsParameters struct {
+	Limit  int
+	Cursor string
+}
+
+// ListTeams returns all workspaces a token can access.
```

Comment:
Please add link to this API's doc in comment.
https://api.slack.com/methods/admin.teams.list

Reply:
done - thanks!